harvard-lts/CURIOSity
# Tab tiles

**To address the lack of ability to tab the tiles in Curiosity's main page*
* * *

# What does this Pull Request do?
Resolution to: LTSMAINT-678, tiles are keyboard inaccessible

# How should this be tested?

Spin up this branch locally, make sure your browser/computer have keyboard accessibility enabled, and make sure you can tab to each tile and also hit enter to go to the collection the tile represents.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
